DFE-Digital/sts-plan-technology-for-your-school
# feat: create 404 page

# feat: create 404 page

## Overview

Addresses ticket [#231874](https://dfe-ssp.visualstudio.com/s190-schools-technology-services/_workitems/edit/231874)


## Changes

Adds 404 not found page in line with the [GDS template](https://design-system.service.gov.uk/patterns/page-not-found-pages)

## How to review the PR

  - Go to a url that doesn't exist
  - Verify that you are redirected to the not found page
  - Verify that other pages function as normal


## Checklist

Delete any rows that do not apply to the PR.
